jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/lambda/go/GoRuntimeGroup.kt

```diff
@@ -22,20 +22,12 @@ class GoRuntimeGroup : SdkBasedRuntimeGroup() {
         LambdaRuntime.GO1_X
     )
 
-    override fun runtimeForSdk(sdk: Sdk): LambdaRuntime? {
-        if (sdk.sdkType !is GoSdkType) {
```

Comment:
Maybe instead of always being GO_1X we should do this then fall back to GO_1X, thoughts?

Reply:
I don't follow

Reply:
Uhhh..... maybe? What are the ramfications

Reply:
Well, it would do nothing different for now. This function is used in 2 paths:
1. picking up runtime when setting up from source file
2. download code from schema
I think we should probably simplify this in the future since it's used in so few places, and isn't really a problem if it's "wrong" but the correct runtime. Until then, I think we are fine just setting `GO1_X` no matter what since it's only called when we already think it's go

Reply:
Makes sense